packages/indexer-client/src/IndexerBaseClient.ts

```diff
@@ -103,8 +103,8 @@ export class IndexerBaseClient {
    * @param params
    */
   async listSubaccounts(
```

Comment:
minor but how about we just call this `getSubaccounts` and rename params/response to just `IndexerQuerySubaccountsParams` / `IndexerQuerySubaccountsResponse`. seems more standard.